impl/src/test/java/org/ehcache/impl/internal/store/heap/ByteSizedOnHeapStoreByValueSPITest.java

```diff
@@ -37,16 +37,25 @@
 import org.ehcache.spi.serialization.Serializer;
 import org.ehcache.spi.service.ServiceConfiguration;
 import org.junit.Before;
+import org.junit.BeforeClass;
 
 import static java.lang.ClassLoader.getSystemClassLoader;
 import static org.ehcache.config.builders.ResourcePoolsBuilder.newResourcePoolsBuilder;
 import static org.ehcache.core.internal.service.ServiceLocator.dependencySet;
+import static org.hamcrest.core.IsNot.not;
+import static org.hamcrest.core.StringStartsWith.startsWith;
+import static org.junit.Assume.assumeThat;
 
 public class ByteSizedOnHeapStoreByValueSPITest extends StoreSPITest<String, String> {
 
   private StoreFactory<String, String> storeFactory;
   private static final int MAGIC_NUM = 500;
 
+  @BeforeClass
+  public static void ignoreInJava9() {
```

Comment:
You're right - this stuff might work now with the very latest EAs (or with `--permit-illegal-access`).  I'll test to be sure.